backend/graminsta/post/urls.py

```diff
@@ -4,11 +4,14 @@
 """
 
 from django.urls import path
-from .views import PostRecordView, FollowView
+from .views import (PostRecordView, FollowView,
+                    UnfollowView, FollowingView)
 
 
 app_name = 'post'
 urlpatterns = [
     path('', PostRecordView.as_view(), name='post'),
     path('follow/', FollowView.as_view(), name='follow'),
-    path('follow/<user_id>/', FollowView.as_view(), name='following_people')]
+    path('unfollow/', UnfollowView.as_view(), name='unfollow'),
```

Comment:
Just a FYI, usually in RESTful API, to remove a following relationship, you can 

```
DELETE /api/follow/5
```

Which means deleting the relationship with id 5

But I think this is OK to compromise  given our deadline.

Reply:
Got it. But does this mean that I will need another view to post the target_user_id and fetch the id of the relationship? Only after fetch the id can I use the delete method, since I do not have the relationship id in frontend, so I chose this simple way to delete for now. I would modify it if I have time later.

Reply:
In my mind, in your followers page, the data you received from backend should be something like:
```
[
{
  id: 1,
  target_user: {
    id: 111,
    first_name: xx
    last_name: yy
  }
},
{
  id: 2,
  target_user: {
    id: 222,
    first_name: xx
    last_name: yy
  }
}
]
```
 Then you should have enough information both for rendering UI and completing the unfollow action.

Reply:
Got it, I think I should modify the serializer to do so.